test_files/declare.sickle.ts

```diff
@@ -1,5 +1,17 @@
-declare module DeclareTest {
+declare module DeclareTestModule {
   export class Foo {
     constructor();
   }
 }
+
+declare interface DeclareTestInterface {
```

Comment:
This is the test for the new "declare" code.  The rest of the changes are whitespace changes due to the refactor.